lib/src/tests/mod.rs

```diff
@@ -0,0 +1,225 @@
+mod arithmetic_methods_class;
+mod assigned_fields_class;
+mod bool_alg_class;
+mod complex_if_class;
+mod empty_class;
+mod empty_method_class;
+mod fib_class;
+mod if_class;
+mod to_java;
+
+use self::to_java::class_to_java;
+use crate::types::Expr::*;
+use crate::types::Stmt::*;
+use crate::types::StmtExpr::*;
+use crate::types::*;
+use std::fs::read;
+use std::fs::File;
+use std::io::Write;
+use std::process::Command;
+use std::process::Stdio;
+
+fn normalize_str(s: std::string::String) -> std::string::String {
+    s.split('\n')
+        .fold("".to_string(), |acc, s| acc + s)
+        .split("\t")
+        .fold("".to_string(), |acc, s| acc + s)
+}
+
+fn stmt_tast_to_ast(stmt: &Stmt) -> Stmt {
+    match stmt {
+        Stmt::TypedStmt(x, typ) => stmt_tast_to_ast(x),
+        Stmt::Block(stmts) => Block(stmts.iter().map(|x| stmt_tast_to_ast(x)).collect()),
+        Stmt::Return(expr) => Return(expr_tast_to_ast(expr)),
+        Stmt::While(cond, body) => While(expr_tast_to_ast(cond), Box::new(stmt_tast_to_ast(body))),
+        Stmt::If(cond, body, elze) => If(
```

Comment:
Why elze?

Reply:
Cause i can't use "else" as the name of a variable, so I just called it "elze" instead